go.mod

```diff
@@ -79,10 +79,10 @@ require (
 	go.uber.org/zap v1.9.1
 	golang.org/x/crypto v0.0.0-20190426145343-a29dc8fdc734
 	golang.org/x/image v0.0.0-20190321063152-3fc05d484e9f
-	golang.org/x/net v0.0.0-20190424112056-4829fb13d2c6
-	golang.org/x/sync v0.0.0-20190423024810-112230192c58 // indirect
+	golang.org/x/net v0.0.0-20190620200207-3b0461eec859
```

Comment:
@jesperhansen17, any idea why these changed as part your commits? I checked upstream, but couldn't find anything from `master`. Perhaps just revert `go.mod` and `go.sum` unless there's a reason to effect a change here?

Reply:
I think I accidentally add a commit on `go.mod` and `go.sum` after I ran some of the makefile commands (which generates new `go.mod` and `go.sum` files for me).

I have reverted the changes.